hasura.planx.uk/tests/utils.js

```diff
@@ -94,6 +94,7 @@ const introspectAs = async (role, userId = undefined) => {
     platformAdmin: gqlWithRole("platformAdmin", userId),
     teamEditor: gqlWithRole("teamEditor", userId),
     api: gqlWithRole("api"),
+    analyst: gqlWithRole("analyst"),
```

Comment:
nit: should this be `gqlWithRole("analyst", userId)` ?

Reply:
Yep good catch - just updated.